test/forest_test.cpp

```diff
@@ -8,13 +8,49 @@
 
 // stlab
 #include <stlab/forest.hpp>
+#include <stlab/forest_algorithms.hpp>
 
 /**************************************************************************************************/
 
 using namespace stlab;
 
 /**************************************************************************************************/
 
+namespace {
+
+/**************************************************************************************************/
+
+template <typename T>
+void print(const stlab::forest<T>& f) {
```

Comment:
stlab is not necessary here, because you have a using namespace at the top.

Reply:
Fixed